docs/code_ref/enhance.rst

```diff
@@ -0,0 +1,7 @@
+*******
+enhance
+*******
+
+This package implements enhancement routines for solar physics data.
```

Comment:
I am not sure I like calling these enhancement routines.

Reply:
Then what should we call them as they work for the entire image

Reply:
Good question, I guess we can leave this for now until I come up with a better name.